pkg/globalvar/globalvar.go

```diff
@@ -0,0 +1,6 @@
+package globalvar
+
+type Message struct {
+	Name          string `json:"name"`
+	PrefetchFiles string `json:"prefetchList"`
```

Comment:
Please make sure the variables express the same meaning. In addition, we often use underscore naming convention for json data.